CREDITS.rst

```diff
@@ -13,7 +13,7 @@ Supporters
 * Development of several features in Thonny 3.0 was proposed and supported by Raspberry Pi Foundation (https://www.raspberrypi.org).
 * Since September 2018, development of Thonny is partially supported by Cybernetica AS (https://cyber.ee).
 
-Serveral companies and organizations have provided us with free hardware for testing Thonny (and for having fun):
```

Comment:
Several companies and organizations have provided us with free hardware for testing Thonny (and for having f